feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/cache/OrchestratorCache.kt

```diff
@@ -54,3 +55,6 @@ internal class OrchestratorCache @Inject constructor(
         }
     }
 }
+
+@Keep
+internal object StepsTypeReference : TypeReference<List<Step>?>()
```

Comment:
Can we keep TypeReference directly so we don't need to do this for each generic type?
https://stackoverflow.com/questions/28433281/how-to-include-typereference-proguard-rule

Reply:
@BurningAXE This is a good idea.
But I don't like the suggested solution in Stackoverflow the `-keepattributes Signature`  The solution will keep all signatures not only Type references

Reply:
I don't see any issue with that - doubt it will increase code side by much. And we don't have anything to hide now :)

Reply:
I used this rule to keep all type reference subclasses 

```
-keep class * extends com.fasterxml.jackson.core.type.TypeReference {
    public <init>(java.lang.reflect.Type);
}
```